cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -213,7 +230,37 @@ data ProtocolParameters =
        --
        -- This is the \"tau\" incentives parameter from the design document.
        --
-       protocolParamTreasuryCut :: Rational
+       protocolParamTreasuryCut :: Rational,
+
+       -- | Cost in ada per word of UTxO storage.
+       --
+       -- /Introduced in Alonzo/
+       protocolParamUTxOCostPerWord :: Maybe Lovelace,
```

Comment:
Hi @dcoutts

Does "word" in this context refer to the size of a [`Data.Word`](http://hackage.haskell.org/package/base-4.15.0.0/docs/Data-Word.html#t:Word)?

(I notice that we still use "byte" elsewhere, for example, in `txFeePerByte`, so I'm guessing the differentiation is deliberate in some way.)

Apologies for the drive-by comment, the PR title piqued my interest and decided to have a look. 😄

Reply:
> In Alonzo, the protocol parameter `minUTxOValue` is deprecated, and replaced by `adaPerUTxOWord`.
This specifies directly the deposit required for storing bytes of data on the ledger in the form of
UTxO entries

It's not clear to me that word means byte, but _I think so_.